### ISSUE TITLE
Crash on startup if last open path was a deleted zip file

### ISSUE THREAD

**Issue by sgynn** [derceg/explorerplusplus#463]
Explorer++ was last open showing the contents of a zip file that has been deleted.
Now every time I try to launch it, it crashes with:
```
Windows cannot open the folder.
Access to the Compressed (zipped) Folder 'C:\Users\someone\Desktop\zipfile.zip' is denied.
```

**To Reproduce**
Steps to reproduce the crash:
1. Create a zip file on the desktop
2. Open the zip file in explorer++
3. Close explorer++
4. Delete the zip file from the desktop
5. Try to open explorer++
4. Explorer++ crashes

**Version**
Explorer++ version 1.5.0.2421 dev (64-bit build)
Build date: May 29 2024 00:58:29
Windows 10

I managed to resolve the issue by recreating the zip file so explorer++ could navigate away from it.

**Comment by derceg** [comment]
Hi sgynn, it sounds like this is probably the same issue described in #453, which was fixed about 2 weeks ago. Could you please check whether the issue is still present in a newer dev version?

**Comment by sgynn** [comment]
Can confirm, issues is fixed in Explorer++ version 1.5.0.2430 dev (64-bit build)
